### ISSUE TITLE
use of "specific" in standard names

### ISSUE THREAD

**Issue by taylor13** [cf-convention/vocabularies#30]
Does anyone know why the word "specific" was included in the standard names land_ice_specific_mass_flux_due_to_calving and land_ice_specific_mass_flux_due_to_calving_and_ice_front_melting (but not, for example, "land_ice_runoff_flux")?

**Comment by efisher008** [comment]
Hi Karl @taylor13 and Jonathan @JonathanGregory,

I'm revisiting this issue as it has been open for over a year with no conclusion reached. 
Did you have any success with contacting the ISMIP community about the changes to the names you proposed? 

Best wishes,
Ellie

**Comment by taylor13** [comment]
I didn't try, and obviously no one responded by commenting on this issue. On the suggestions that Jonathan and I have expressed agreement, perhaps action can now be taken.  My quick read of the above says a few issues could probably benefit from more thought/input.  Do you (@efisher008) want to talk a crack at summarizing, or is the input too muddled for that?

**Comment by efisher008** [comment]
Hi Karl @taylor13, 

I will have to look through the issue carefully but will try my best to summarise. Please bear with me while I do this!

Best,
Ellie

**Comment by taylor13** [comment]
Thanks for consulting with Sophie.  I agree with the changes you suggest immediately above.

**Comment by efisher008** [comment]
Hi @JonathanGregory,

I have a question about the use of `amount` in the names suggested above. Is this defined according to the CF standard name construction guidelines (i.e. units of **kg m-2**)?

Best,
Ellie

**Comment by JonathanGregory** [comment]
Yes, `land_ice_amount` is intended to mean `kg m-2`. This may be misleading in the context of land ice, though. Perhaps  `tendency_of_land_ice_mass_per_unit_area_due_to_calving` would be better.

**Comment by efisher008** [comment]
Hi @JonathanGregory @taylor13,

I think given that `amount` has a defined unit in CF, it should be sufficient to call these names `tendency_of_land_ice_amount_due_to_calving` etc. I have made [these changes](https://github.com/cf-convention/vocabularies/issues/30#issuecomment-2286247778) in the editor and the names are currently listed as "under discussion". 

- `tendency_of_land_ice_amount_due_to_calving` - https://cfeditor.ceda.ac.uk/proposal/5421/edit
- `tendency_of_land_ice_amount_due_to_calving_and_ice_front_melting` - https://cfeditor.ceda.ac.uk/proposal/5422/edit
- `land_ice_surface_mass_balance_flux` - https://cfeditor.ceda.ac.uk/proposal/5423/edit
- `land_ice_lwe_surface_mass_balance_rate` - https://cfeditor.ceda.ac.uk/proposal/5424/edit
- `land_ice_basal_mass_balance_flux` - https://cfeditor.ceda.ac.uk/proposal/5425/edit
- `tendency_of_land_ice_amount` - https://cfeditor.ceda.ac.uk/proposal/5420/edit

I will address Karl's [earlier comments](https://github.com/cf-convention/vocabularies/issues/30#issuecomment-2255955984) in a separate post. 

Best,
Ellie

**Comment by taylor13** [comment]
maybe "amount" is o.k., but is "mass_balance" well understood?  It reminds me of a mechanical scale used to weigh objects by matching their mass with a known masses.  What is proposed for defining what is meant by "mass_balance", or is it so obvious to everyone but me that no further explanation is needed.  (Don't be afraid to say so, if it is.)

**Comment by JonathanGregory** [comment]
Dear Karl

"Mass balance" is well-understood in glaciology, but that may not be sufficient for standard names. It's in the [UNESCO glaciological glossary](https://wgms.ch/downloads/Cogley_etal_2011.pdf) and the [IPCC WG1 AR6 glossary](https://www.ipcc.ch/report/ar6/wg1/downloads/report/IPCC_AR6_WGI_AnnexVII.pdf), for instance. It means the difference between accumulation (addition of mass) and ablation (loss of mass). "Mass budget" is a synonym, less commonly used in the literature. Do you think "mass budget" would be preferable for standard names?

**Comment by taylor13** [comment]
Perhaps, then, "mass balance" is o.k., but it may not be obvious what the sign convention is, so could the description include something like "Mass balance is a measure of the change in ice mass over time and is taken to be positive when the addition of mass (accumulation) exceeds loss (ablation)."

**Comment by JonathanGregory** [comment]
Dear Karl @taylor13, Ellie @efisher008

The standard names we're discussing are renamings, rather than new entries. The existing descriptions of the two standard names containing `specific_mass_balance_flux`, either `surface` or `basal`, contain the text

> "Specific mass balance" means the net rate at which ice is added per unit area. A negative value means loss of ice.

For the renamed quantities (deleting `specific`), I think this should be changed to

> "Mass balance flux" means _etc. unchanged_

The descriptions of the two standard names containing `surface_specific_mass_balance_rate` current includes

> Specific mass balance means the net rate at which ice is added per unit area at the land ice surface due to all processes of surface accumulation and ablation. A negative value means loss of ice.

which should be changed to

> "Surface mass balance" means _etc. unchanged_

The descriptions of the two `tendencies` are OK as they stand.

Ellie, I can't view the entries with the links you gave. They give

```
You need to be logged in to see this page. Use the link below to login.
Login
```

and if I click `Login`, I get

```
403 Forbidden
nginx
```

Best wishes

Jonathan

**Comment by efisher008** [comment]
Hi Jonathan @JonathanGregory,

That is interesting - I've been including these links as standard practice in GitHub issues for a while, and nobody has said before that they cannot actually access the names! 

It seems the 'edit' suffix in the link URL requires an admin login. Please try using the links without this last 'edit', i.e. https://cfeditor.ceda.ac.uk/proposal/5421/. This should bring you into 'view' mode. I tested this by logging out of the admin interface and then trying the new link, so I think it should work.

Best wishes,
Ellie

**Comment by efisher008** [comment]
I also noticed that there was a `mass_balance_rate` name missing from [this comment](https://github.com/cf-convention/vocabularies/issues/30#issuecomment-2286247778) which @JonathanGregory refers later: `land_ice_surface_mass_balance_rate`.

I have added it here: https://cfeditor.ceda.ac.uk/proposal/5426/ 

PS. I have also changed (shortened) the CF editor links so they should take you to view the entry, not edit it.

**Comment by JonathanGregory** [comment]
Thanks for adjusting the links. These don't work either for me, unfortunately. They give
```
Not Found
The requested resource was not found on this server.
```

**Comment by taylor13** [comment]
The modified to the descriptions suggested by Jonathan in https://github.com/cf-convention/vocabularies/issues/30#issuecomment-2298547175 are now clear to me.  Thanks!
